SafeguardPasswords/Scripts/Bootstrap.ps1

```diff
@@ -0,0 +1,79 @@
+$ErrorActionPreference = "Stop"

```

Comment:
#Note to self and the devs: We have to think of a way in the future to not download the binaries redundantly for all connectors (Common, Sessions, Passwords) but keep the configs separately.